docs/examples/collapse/custom-icon.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="demo-collapse">
+    <el-collapse v-model="activeNames" @change="handleChange">
+      <el-collapse-item title="Consistency" name="1" :icon="CaretRight">
+        <div>
+          Consistent with real life: in line with the process and logic of real
+          life, and comply with languages and habits that the users are used to;
+        </div>
+        <div>
+          Consistent within interface: all elements should be consistent, such
+          as: design style, icons and texts, position of elements, etc.
+        </div>
+      </el-collapse-item>
+      <el-collapse-item title="Feedback" name="2">
+        <template #icon="{ isActive }">
+          <span class="icon-ele">{{ isActive ? '已展开' : '已收起' }}</span>
```

Comment:
It is best to show English here to be consistent with other examples.

![image](https://github.com/user-attachments/assets/15e58c9d-237d-424c-972e-7109b7ed01fb)